src/hooks/useOrderList.ts

```diff
@@ -57,9 +57,18 @@ export function useOrderList() {
 
   const openOrders = useMemo(
     () =>
-      orders.filter(
-        (order) => order.singleOrder && order.status !== "fulfilled",
-      ),
+      orders
+        ?.filter((order) => order.singleOrder && order.status !== "fulfilled")
+        .map((order) => {
+          const isCancelling =
+            localStorage.getItem(`status-${order.id}`) === `cancelling`;
```

Comment:
why use this from localstorage?

Reply:
to preserve the status between reloads

Reply:
but shouldn't we instead commit the whole thing to local storage?

Reply:
humm maybe, I think that you change it, right? I will take a look on it